packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/petrinet/Ontology.java

```diff
@@ -0,0 +1,30 @@
+package software.uncharted.terarium.hmiserver.models.petrinet;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.smallrye.jwt.config.ConfigLogging.log;
+
+@Data
+public class Ontology implements Serializable {
+	private final String name;
+	private final String curie;
+	private String title;
+	private String description;
+	private String link;
+
+	public Ontology (String input) {
+		final Matcher matcher = Pattern.compile("\\(\\'(.+?)\\', \\'(.+?)\\'\\)").matcher(input);
```

Comment:
This may be worth a comment on what you're looking to do - if we hadn't spoke yesterday this would leave me scratching my head :D